### PR TITLE
Add com.moura.Progress

### PR DESCRIPTION
**Pull request by smolBlackCat**
### Please confirm your submission meets all the criteria

Progress is a simple task list app utilising the kanban style

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ ] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: .github/actions/flatpak-external-data-checker/Dockerfile
Comment (hfiguiere) [inline .github/actions/flatpak-external-data-checker/Dockerfile:1]: the .github directory doesn't belong here.

File: CONTRIBUTING.md
Comment (hfiguiere) [inline CONTRIBUTING.md:1]: this is not part of the `new-pr` branch either.

Comment (smolBlackCat) [reply CONTRIBUTING.md:1]: Sorry, I must have misconfigured something after I made the pull request

File: COPYING
Comment (hfiguiere) [inline COPYING:1]: this is not part of the new-pr branch either.

File: README.md
Comment (hfiguiere) [inline README.md:1]: this is not part of the new-pr branch either.

File: com.moura.Progress.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
+{
+    "id": "com.moura.Progress",
+    "runtime": "org.gnome.Platform",
+    "sdk": "org.gnome.Sdk",
+    "runtime-version": "45",
```
Comment (hfiguiere) [inline com.moura.Progress.json:5]: 46 is out

File: com.moura.Progress.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    "command": "progress-tracker",
+    "modules": [
+        {
+            "name": "gtkmm-4-10",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dmaintainer-mode=false"
+            ],
+            "cleanup": [
+		"*.a",
```
Comment (hfiguiere) [inline com.moura.Progress.json:15]: the indentation is weird making things hard to read.

File: com.moura.Progress.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/smolBlackCat/progress-tracker.git",
+                    "tag": "v1.0"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```
Comment (hfiguiere) [inline com.moura.Progress.json:146]: finish args should be above modules, as expected.

File: com.moura.Progress.json [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+        "--filesystem=home",
```
Comment (hfiguiere) [inline com.moura.Progress.json:147]: why is 'home' filesystem access needed?

File: CONTRIBUTING.md
Comment (hfiguiere) [reply CONTRIBUTING.md:1]: like the 30 unnecessary commits?

**Pull request closed without merging**